make/ide/eclipse/CreateWorkspace.gmk

```diff
@@ -0,0 +1,392 @@
+#
+# Copyright (c) 2016, 2022, Oracle and/or its affiliates. All rights reserved.
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.  Oracle designates this
+# particular file as subject to the "Classpath" exception as provided
+# by Oracle in the LICENSE file that accompanied this code.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+# or visit www.oracle.com if you need additional information or have any
+# questions.
+#
+
+default: all
+
+include $(SPEC)
+include MakeBase.gmk
+include TextFileProcessing.gmk
+include Utils.gmk
+include Modules.gmk
+
+JVM_VARIANT := $(JVM_VARIANT_MAIN)
+include hotspot/HotspotCommon.gmk
+include hotspot/lib/JvmFeatures.gmk
+include hotspot/lib/JvmOverrideFiles.gmk
+include hotspot/lib/JvmFlags.gmk
+
+# Warning: This file does not have the best formatting!
```

Comment:
Is this still true? If so, we should fix that so the comment can be removed.

After having read through the file. I find quite a few places where broken up lines have 2 space indentation instead of 4. (See https://openjdk.org/groups/build/doc/code-conventions.html) Fixing this should be pretty quick and simple and will greatly improve readability at least for me and Magnus. I will mark some of the typical cases for you.

Reply:
Not anymore, at least I would hope not. This was back when everything in the file was completely generated by handwritten printing and is not up to date any longer

Reply:
I found just one more missed indentation. It looks much better to me now. Could you also remove this comment, as it's not relevant anymore.